### PR TITLE
Add abb_resources package from abb_experimental repo

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

File: abb/package.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
   <run_depend>abb_driver</run_depend>
+  <run_depend>abb_resources</run_depend>
```
Comment (gavanderhoorn) [inline abb/package.xml]: It would be nice if we can keep this list alphabetically sorted.

File: abb_resources/package.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      ABB related packages.
+    </p>
+  </description>
+  <author email="levi.armstrong@swri.org">Levi Armstrong (Southwest Research Institute)</author>
+  <maintainer email="sedewards@swri.org">Shaun Edwards (Southwest Research Institute)</maintainer>
+  <license>Apache2</license>
+
+  <url type="website">http://wiki.ros.org/abb_resources</url>
+  <url type="bugtracker">https://github.com/ros-industrial/abb_experimental/issues</url>
+  <url type="repository">https://github.com/ros-industrial/abb_experimental</url>
```
Comment (gavanderhoorn) [inline abb_resources/package.xml]: This package is no longer in `abb_experimental`.

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]

**Comment by gavanderhoorn** [comment]
And also please update the commit msg to include a reference to ros-industrial/abb_experimental@f3816a1.

**Comment by Levi-Armstrong** [comment]
>And also please update the commit msg to include a reference to ros-industrial/abb_experimental@f3816a1.

Just so I understand you want me to change the message 
from:
- Add abb_resources package from abb_experimental repo

to:
- Add abb_resources package from ros-industrial/abb_experimental@f3816a1.

**Comment by gavanderhoorn** [comment]
No. I'd like to see the commit msg of 371aa4e (in this PR) to be something like:

```
Add abb_resources package from abb_experimental repo.

Removed from abb_experimental repository in ros-industrial/abb_experimental@f3816a1.
```

**Comment by gavanderhoorn** [comment]
That way ppl who want to can track the move across the different repositories.

**Comment by gavanderhoorn** [comment]
O wait, you edited your comment while I was commenting.

That is also good.

**Comment by Levi-Armstrong** [comment]
Sorry, I hit the mouse and it submitted it before I was finished.

**Comment by Levi-Armstrong** [comment]
Ok, I will update the message.

**Comment by Levi-Armstrong** [comment]
@gavanderhoorn, I updated the package.xml and change the commit message.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
If you could also update the travis config over at `abb_experimental` that would be great.

**Pull request merged**